assets/queries/openAPI/general/api_key_exposed_in_operation_security/query.rego

```diff
@@ -4,16 +4,18 @@ import data.generic.openapi as openapi_lib
 
 CxPolicy[result] {
 	doc := input.document[i]
-	openapi_lib.check_openapi(doc) != "undefined"
+	version := openapi_lib.check_openapi(doc)
+	version != "undefined"
 
 	security := doc.paths[path][operation].security[x][s]
-	doc.components.securitySchemes[s].type == "apiKey"
+	openapi_lib.api_key_exposed(doc, version, s)
```

Comment:
I think, here, it is checking global `security`, no? Also, where are you using `security` variable?

Reply:
No, because we are checking the security `s` defined in operations